detections/application/splunk_cmd_injection_using_external_lookups.yml

```diff
@@ -0,0 +1,49 @@
+name: Splunk Cmd Injection Using External Lookups
+id: 901ae2e2-b47b-4adc-b296-a151a8fccdca
+version: 1
+date: '2023-09-05'
+author: Rod Soto
+status: production
+type: Hunting
+data_source:
+- audittrial
+description: In Splunk Enterprise versions below 8.2.12, 9.0.6, and 9.1.1, an attacker can create an external lookup that calls a legacy internal function. The attacker can use this internal function to insert code into the Splunk platform installation directory. From there, a user can execute arbitrary code on the Splunk platform Instance.
+search: '`audit_searches` search = "| *runshellscript* """ | stats count min(_time) as firstTime max(_time) as lastTime by search user splunk_server 
```

Comment:
Lets rename `host` to `dest` and update this on observables and message and this search is currently failing CI

Reply:
ignore

Reply:
After followup conversation, we will keep this as splunk_server for these detections.